wom/src/main/scala/wdl4s/wdl/AstTools.scala

```diff
@@ -305,6 +305,8 @@ object AstTools {
         && a.getAttribute("lhs") == terminal
         && a.getAttribute("rhs").isTerminal => a.getAttribute("rhs").asInstanceOf[Terminal]
     }
+
+    def fullVariableReferenceString = terminal.getSourceString + (terminalSubIdentifier map { "." + _.getSourceString } getOrElse "")
```

Comment:
I did that in the previous commit but @geoffjentry and @cjllanwarne asked me to knock it off so I did.